File: exercises/csharp/TriviaGame/TriviaGame/Player.cs
using System;
using System.Collections.Generic;

namespace TriviaGame
{
    public class Player
    {
        public List<string> totalPlayers;
        public int[] playerBoardPositionState = new int[6];
        public int[] playerPurseTotalState = new int[6];

        public bool[] playerPenaltyBoxState = new bool[6];

        public bool AddPlayer(string playerName)
        {
            totalPlayers.Add(playerName);
            playerBoardPositionState[GetTotalPlayersCount()] = 0;
            playerPurseTotalState[GetTotalPlayersCount()] = 0;
            playerPenaltyBoxState[GetTotalPlayersCount()] = false;
            PrintNewPlayerDetails(playerName);
            return true;
        }

        private void PrintNewPlayerDetails(string playerName)
        {
            Console.WriteLine(playerName + " was added");
            Console.WriteLine("They are player number " + totalPlayers.Count);
        }

        public int GetTotalPlayersCount()
        {
            return totalPlayers.Count;
        }

    }
}

File: exercises/java/src/main/java/com/adaptionsoft/games/categories/Player.java
package com.adaptionsoft.games.categories;

public class Player {
    String playerName;
    Integer boardLocation;
    Integer purse;
    Boolean isInPenaltyBox;
    Boolean isGettingOutOfPenaltyBox;

    public Player(String playerName, Integer playerNumber) {
        this.playerName = playerName;
        this.boardLocation = 1;
        this.purse = 0;
        this.isInPenaltyBox = false;
        this.isGettingOutOfPenaltyBox = false;

        System.out.println(playerName + " was added");
        System.out.println("They are player number " + playerNumber);
    }
}
